# basic modes
uint8 PX4_MODE_STANDBY = 0
uint8 PX4_MODE_STARTING = 1
uint8 PX4_MODE_FINISHED = 2
uint8 PX4_MODE_STOPPED = 3

uint8 PX4_CMD_TAKEOFF = 4
uint8 PX4_CMD_LANDING = 5
uint8 PX4_MODE_OFFBOARD = 6
uint8 PX4_SET_ROTOR = 7
uint8 PX4_SET_MISSION = 8
uint8 PX4_SET_HOLD = 9

bool set_rotor
uint8 mode_req # filled by MAV_MODE enum value or 0 if custom_mode != ''
---
bool rotor_state
bool success
